examples/provider_test.go

```diff
@@ -51,13 +51,14 @@ func TestV3HTTPProvider(t *testing.T) {
 	// Verify the Provider with local Pact Files
 	err := verifier.VerifyProvider(t, provider.VerifyRequest{
 		ProviderBaseURL: "http://127.0.0.1:8111",
-		Provider:        "V3Provider",
+		Provider:        "V4Provider",
 		ProviderVersion: os.Getenv("APP_SHA"),
-		BrokerURL:       os.Getenv("PACT_BROKER_BASE_URL"),
-		PactFiles: []string{
-			filepath.ToSlash(fmt.Sprintf("%s/PactGoV3Consumer-V3Provider.json", pactDir)),
-			filepath.ToSlash(fmt.Sprintf("%s/PactGoV2ConsumerMatch-V2ProviderMatch.json", pactDir)),
-		},
+		// BrokerURL:       os.Getenv("PACT_BROKER_BASE_URL"),
```

Comment:
Did you mean to leave these uncommented?

Reply:
Yeah, I think these changes crept in from various manual testing, I'll go through and clean them up tomorrow.